HACKING.jst.md

```diff
@@ -172,14 +176,10 @@ is:
   when running your test (uncommon) write that output to
   `tests/passing/tests/foo.ml.err`.
 
-Now, run `dune test`. It will discover your new file and suggest edits to
-the generated `tests/passing/dune.inc` file to run your new tests. Run
-`dune promote` to update `dune.inc`. This will *not* accept your new tests -- it
```

Comment:
Does `make test` really cover all of this? That is, I had to promote *twice* when adding a new test: once to accept the changes to `dune.inc` and the next to actually accept the new tests. I haven't experimented, but it looks like `make test` will not quite do this all in one go. Regardless, I think it's helpful to learn here that `dune.inc` is generated and how it gets updated.